net/modemmanager/files/modemmanager.proto

```diff
@@ -397,7 +399,15 @@ proto_modemmanager_setup() {
 	echo "starting connection with apn '${apn}'..."
 	proto_notify_error "${interface}" MM_CONNECT_IN_PROGRESS
 
-	connectargs="apn=${apn}${iptype:+,ip-type=${iptype}}${cliauth:+,allowed-auth=${cliauth}}${username:+,user=${username}}${password:+,password=${password}}${pincode:+,pin=${pincode}}"
+	# setup allow-roaming parameter
+	if [ -n "${allow_roaming}" ] && [ "${allow_roaming}" -eq 0 ];then
+		allow_roaming="no"
+	else
+		# allowed unless a user set the opposite
+		allow_roaming="yes"
+	fi
+
+	connectargs="apn=${apn}${iptype:+,ip-type=${iptype}}${cliauth:+,allowed-auth=${cliauth}}${username:+,user=${username}}${password:+,password=${password}}${pincode:+,pin=${pincode}},allow-roaming=${allow_roaming}"
```

Comment:
Maybe move the `allow-roaming` key/value pair to after `apn` so that all keys that are always set are given at the beginning?